tests/lib/rules/jsx-curly-brace-presence.js

```diff
@@ -96,7 +96,8 @@ ruleTester.run('jsx-curly-brace-presence', rule, {
       options: [{props: 'never'}]
     },
     {
-      code: '<App>{<myApp></myApp>}</App>'
+      code: '<App>{<myApp></myApp>}</App>',
```

Comment:
this arguably is a breaking change :-/ hopefully we can consider this a patch though.